CONTRIBUTING.md

```diff
@@ -309,6 +309,25 @@ about:
     when someone else is asking for feedback on their code, and universe balances out.
 
 
+Backporting
+-----------
+
+After the original PR was merged into master, backporting is done only
+when necessary - the fix either needs to be present in a release branch
+(e.g. a security issue) or on request.
+If the backport is trivial and can be cherry-picked automatically,
+just request a "Needs backport" tag for the original PR.
+If backport needs any conflict resolution, create the PR manually and
+wait for comments.
+
+The commit message of a backport should additionally contain following
+two lines in the end:
+
+```
+Github-Pull: #<original issue/PR number>
+Rebased-From: <sha checksum of the code already merged into master branch>
```

Comment:
commit hash of the respective merged commit?

Reply:
Fixed in 842aa85aa ...